data/data_loader.py

```diff
@@ -0,0 +1,43 @@
+
+from utils.config import top_config
+from torch.utils.data import Dataset, DataLoader
+import torch
+
+import os
```

Comment:
According to PEP8:

"Imports should be grouped in the following order:

1. Standard library imports.

2. Related third party imports.

3. Local application/library specific imports.

You should put a blank line between each group of imports."

In our case here, `os` and `sys` belong to standard lib, `torch` is third-party, and `utils` is local application/lib.

Reply:
Done

Reply:
`json` seems also third-party to me.